sale_margin_delivered/views/sale_margin_delivered_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale_margin.sale_margin_sale_order_line_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='margin']" position="after">
+                <field
+                    name="purchase_price_delivery"
+                    string="Cost Price dlvd."
+                    optional="hide"
+                />
+                <field
+                    name="margin_delivered_percent"
+                    string="% Margin"
```

Comment:
```suggestion
                    string="% Margin dlvd. (%)"
```



This string is error prone IMO. The standard margin (introduced in sale_margin) is name "Margin (%)". 

if both fields are displayed, it is not possible for end user to know what are the field displayed. In my opinion, should "Margin dlvd. (%)"
![image](https://github.com/OCA/margin-analysis/assets/3407482/26751829-393b-4bc7-9f6e-170fc74200cc)

Reply:
You are right much clearer and avoids leading to errors, thanks for everything :)